packages/repack/src/plugins/ModuleFederationPluginV2.ts

```diff
@@ -83,11 +85,14 @@ export interface ModuleFederationPluginV2Config
 export class ModuleFederationPluginV2 implements RspackPluginInstance {
   public config: MF.ModuleFederationPluginOptions;
   private deepImports: boolean;
+  private disableDefaultRuntimePlugins: boolean;
 
   constructor(pluginConfig: ModuleFederationPluginV2Config) {
-    const { reactNativeDeepImports, ...config } = pluginConfig;
+    const { disableDefaultRuntimePlugins, reactNativeDeepImports, ...config } =
+      pluginConfig;
     this.config = config;
     this.deepImports = reactNativeDeepImports ?? true;
+    this.disableDefaultRuntimePlugins = disableDefaultRuntimePlugins ?? false;
```

Comment:
Not a fan of netagtions in publicAPI config. Consider changing this to `defaultRuntimePlugins: string[]` and allow user to override this with e.g. empty array in case they don't want default.

Reply:
Good point, let's go with that 👍

Reply:
updated the PR with the changes we talked about in here 👍